shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/STS/Newpp.hs

```diff
@@ -52,7 +52,7 @@ instance STS (NEWPP crypto) where
   type Environment (NEWPP crypto) = NewppEnv crypto
   type BaseM (NEWPP crypto) = ShelleyBase
   data PredicateFailure (NEWPP crypto)
-    = UnexpectedDepositPot
+    = UnexpectedDepositPot Coin Coin
```

Comment:
Maybe nice to add some more doc indicating what these fields represent

Reply:
this is a great suggestion! if it sounds good to you, I'll add comments to all the STS errors in a follow up PR (and try to get the last few remaining ones that are still lacking proper details fleshed out).